### ISSUE TITLE
NFO file and thumbnail generation

### ISSUE THREAD

**Issue by jctennis** [ZGorlock/YoutubeDownloader#5]
Would it be possible to generate a nfo file and thumbnail image for the downloaded videos to make it easier for Jellyfin or something like that to pick uo?

**Comment by jctennis** [comment]
Even if the .json files could be generated I could probably use this project to convert them to .nfo https://github.com/owdevel/ytdl-nfo

**Comment by ZGorlock** [comment]
Yes this is possible, there is work for similar functionality planned for after I finish a couple other tasks. I will include this when doing that work.

For now, if you would like the thumbnails at least, you can put something similar to this in your ChannelProcesses.java:
`BaseProcess.forEach(channel, videoMap, (id, video) -> {
    final File thumbnail = new File(Project.TMP_DIR, (id + ".png"));
    if (!thumbnail.exists()) {
        Internet.downloadFile(video.getInfo().getThumbnails().getBest().getUrl(), thumbnail);
    }
});`
Note that, currently, saving them in the video output directory will cause them to be deleted if keepClean is set to true for the Channel.

You may even be able to create the nfo file you need; all other information should be available in that VideoInfo object.
